tests/misc/sram_regions/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
nit: test name could embed the "arm" + "mpu" strings so it is more visible that ARM MPU attrs are tested.

Reply:
yeah, I'll use `arm_mpu_regions`